shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -1145,14 +1145,13 @@ - (BOOL)resignFirstResponder {
 }
 
 - (BOOL)canPerformAction:(SEL)action withSender:(id)sender {
-  // When scribble is available, the FlutterTextInputView will display the native toolbar unless
```

Comment:
what is a "native toolbar"? i assume it's not the edit menu?

Reply:
From my memory I think the system context menu (AKA text selection toolbar) was showing up? Maybe @fbcouch can confirm.

I did not see that happen just now when I tested Scribble on an iPad running iOS 17.